test/JDBC/expected/sp_xml_preparedocument-vu-verify.out

```diff
@@ -0,0 +1,495 @@
+
```

Comment:
Do we see possibility of adding testcase for following scenario?
Current user tries dropping internally created temp table.

Changes looks good overall!